doc/packaging.md

```diff
@@ -0,0 +1,334 @@
+```
+</td>
+<td>
+
+```
+/var/lib/aziot/
+└── edged/
+```
+</td>
+<td>
+
+```
+/var/lib/aziot/
+├── certd/
+│   └── certs/
+├── identityd/
+└── keyd/
+    └── keys/
+```
+</td>
+</tr>
+<tr>
+<td>Package dependencies</td>
+<td>
+
+- `moby-engine`
+- `openssl`
+</td>
+<td>
+
+- `aziot-identity-service`
+- `moby-engine`
+- `openssl`
+</td>
+<td>
+
+- `openssl`
+</td>
+</tr>
+</tbody>
+</table>
+
+## Installation
+
+For full documentation refer to the official documentation to [Install or uninstall Azure IoT Edge for Linux](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11). What follows is a brief summary with accompanying notes.
+
+```sh
+apt install aziot-edge
+
+sudo cp /etc/aziot/config.toml.edge.template /etc/aziot/config.toml
+
+sudo nano -w /etc/aziot/config.toml
+
+iotedge config apply
+```
+
+After installing the `aziot-edge` package, copy the `/etc/aziot/config.toml.edge.template` to `/etc/aziot/config.toml` and edit the file to provide provisioning information.  Then run `iotedge config apply` to apply the configuration to the services. This performs initialization for both the IS+KS+CS+TPMS components installed by the `aziot-identity-service` package and the MR component installed by the `aziot-edge` package.
+
+## Updating from `iotedge` to `aziot-edge`
+
+```sh
+apt remove iotedge libiothsm-std
+
+apt install aziot-edge
+
+sudo iotedge config import
+```
+
+The `iotedge config import` can be used to generate the `/etc/aziot/config.toml` based on the old `/etc/iotedge/config.yaml` left behind after uninstalling versions 1.1 or earlier of IoT Edge. This assumes that the `--purge` flag is not used to remove packages as that would remove the old configuration.
+
+We do not support simultaneously running the services from both the old and new packages. They would step over each other trying to provision the device and manage Docker modules. We enforce mutual exclusivity between the packages by having them conflict with each other. This causes the distribution's package manager (e.g `apt` or `apt-get`) to not allow them both to be installed at the same time. This does not work if `dpkg` is used directly. Instead, the existing `iotedge` and `libiothsm-std` packages must be removed before installing the `aziot-edge` package (or even the `aziot-identity-service` package).
+
+### Importing Configuration
+
+This is provided as a convenience to ease the transition from v1.1 to v1.2.
+
+```sh
+iotedge config import
+```
+
+Notes on behavior:
+
+- The device provisioning method is parsed from `/etc/iotedge/config.yaml` and translated into the provisioning information in `/etc/aziot/config.toml`.
+
+- User-provided certificates like device ID, device CA and trust bundle, and their corresponding private key files, will be added as preloaded keys and certs in `config.toml`. The files themselves will not be moved, because they are managed by the user rather than belonging in our services' directories.
```

Comment:
Would be to good to mention either in this point or a new one that the imported device CA will now be referred to as the "Edge CA".

Reply:
Added as a new point below this one